src/us/kbase/workspace/test/kbase/KBaseWorkspaceConfigTest.java

```diff
@@ -433,41 +581,41 @@ public void configFailNullRequiredEntries() throws Exception {
 				"Missing listener class: listener-listener1-class",
 				"Invalid listener configuration item: listener-listener2-config-");
 		
-		final KBaseWorkspaceConfig kwc = new KBaseWorkspaceConfig(cfg);
+		final String paramReport = "mongodb-user=user\nsample-service-url=https://foo.com\n";
 		
-		assertThat("incorrect admin read roles", kwc.getAdminReadOnlyRoles(), is(set()));
-		assertThat("incorrect admin roles", kwc.getAdminRoles(), is(set()));
-		assertThat("incorrect auth2 url", kwc.getAuth2URL(), nullValue());
-		assertThat("incorrect auth url", kwc.getAuthURL(), nullValue());
-		assertThat("incorrect backend token", kwc.getBackendToken(), nullValue());
-		assertThat("incorrect db", kwc.getDBname(), nullValue());
-		assertThat("incorrect errors", kwc.getErrors(), is(errors));
-		assertThat("incorrect srvc token", kwc.getHandleServiceToken(), nullValue());
-		assertThat("incorrect backend type", kwc.getBackendType(), nullValue());
-		assertThat("incorrect backend url", kwc.getBackendURL(), nullValue());
-		assertThat("incorrect backend user", kwc.getBackendUser(), nullValue());
-		assertThat("incorrect backend container", kwc.getBackendContainer(), nullValue());
-		assertThat("incorrect backend region", kwc.getBackendRegion(), nullValue());
-		assertThat("incorrect backend trust certs", kwc.getBackendTrustAllCerts(), is(false));
-		assertThat("incorrect srvc url", kwc.getHandleServiceURL(), nullValue());
-		assertThat("incorrect host", kwc.getHost(), nullValue());
-		assertThat("incorrect info msgs", kwc.getInfoMessages(), is(Collections.emptyList()));
-		assertThat("incorrect listeners", kwc.getListenerConfigs(), is(Collections.emptyList()));
-		assertThat("incorrect mongo pwd", kwc.getMongoPassword(), nullValue());
-		assertThat("incorrect mongo user", kwc.getMongoUser(), nullValue());
-		assertThat("incorrect param report", kwc.getParamReport(),
-				is("mongodb-user=user\nsample-service-url=https://foo.com\n"));
-		assertThat("incorrect bytestream token", kwc.getBytestreamToken(), nullValue());
-		assertThat("incorrect bytestream url", kwc.getBytestreamURL(), nullValue());
-		assertThat("incorrect bytestream user", kwc.getBytestreamUser(), nullValue());
-		assertThat("incorrect sample token", kwc.getSampleServiceToken(), nullValue());
-		assertThat("incorrect sample url", kwc.getSampleServiceURL(),
-				is(new URL("https://foo.com")));
-		assertThat("incorrect temp dir", kwc.getTempDir(), nullValue());
-		assertThat("incorrect type db", kwc.getTypeDBName(), nullValue());
-		assertThat("incorrect ws admin", kwc.getWorkspaceAdmin(), nullValue());
-		assertThat("incorrect has err", kwc.hasErrors(), is(true));
-		assertThat("incorrect ignore hs", kwc.ignoreHandleService(), is(false));
+		assertConfigCorrect(
```

Comment:
Watch out for this one, the asserts aren't in quite the same order as the rest of the tests